var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -7,57 +7,61 @@
 
 
 class Adios2(CMakePackage):
-    """Next generation of ADIOS developed in the Exascale Computing Program"""
+    """The Adaptable Input Output System version 2,
+    developed in the Exascale Computing Program"""
 
-    homepage = "https://www.olcf.ornl.gov/center-projects/adios/"
-    url      = "https://github.com/ornladios/ADIOS2/archive/v2.0.0.tar.gz"
-    git      = "https://github.com/ornladios/ADIOS2.git"
+    homepage = "https://csmd.ornl.gov/software/adios2"
+    url = "https://github.com/ornladios/ADIOS2/archive/v2.5.0.tar.gz"
+    git = "https://github.com/ornladios/ADIOS2.git"
 
-    maintainers = ['ax3l', 'chuckatkins']
+    maintainers = ['ax3l', 'chuckatkins', 'williamfgc']
 
     version('develop', branch='master')
-    version('2.5.0', sha256='7c8ff3bf5441dd662806df9650c56a669359cb0185ea232ecb3578de7b065329')
-    version('2.4.0', sha256='50ecea04b1e41c88835b4b3fd4e7bf0a0a2a3129855c9cc4ba6cf6a1575106e2')
-    version('2.3.1', sha256='3bf81ccc20a7f2715935349336a76ba4c8402355e1dc3848fcd6f4c3c5931893')
-    version('2.2.0', sha256='77058ea2ff7224dc02ea519733de42d89112cf21ffe7474fb2fa3c5696152948')
-    version('2.1.0', sha256='d4df3b3035b4236b51c77b59d68e756e825834b2ea3cb17439927a027831453b')
-    version('2.0.0', sha256='4eeedf4404824d8de6e4ef580b8a703c0aedb5c03f900f5ce6f85f0b35980135')
-
+    version(
+        '2.5.0', sha256='7c8ff3bf5441dd662806df9650c56a669359cb0185ea232ecb3578de7b065329')
+    version(
+        '2.4.0', sha256='50ecea04b1e41c88835b4b3fd4e7bf0a0a2a3129855c9cc4ba6cf6a1575106e2')
+    version(
+        '2.3.1', sha256='3bf81ccc20a7f2715935349336a76ba4c8402355e1dc3848fcd6f4c3c5931893')
+
+    # general build options
+    variant('mpi', default=True, description='Enable MPI')
+    variant('build_type', default='Release',
```

Comment:
Spack has a built0in variant for all cmake packages that already handles this so the explicit `build_type` variant here can be removed.

Reply:
The difference is that this variant defaults to `Release`, whereas the one in CMakePackage defaults to `RelWithDebInfo`. So they're not exactly identical.

Reply:
@chuckatkins see @adamjstewart comment, I'm trying to keep it 1:1 with our cmake build. Thanks.